.github/actions/paths-filter/action.yml

```diff
@@ -40,6 +40,14 @@ outputs:
         steps.filter-common.outputs.frontend-change == 'true' ||
         steps.filter-identity.outputs.identity-frontend-change == 'true'
       }}
+  frontend-change:
+    description: Output whether any frontend has changed
+    value: >-
+      ${{
+        github.event_name == 'push' ||
+        steps.filter-common.outputs.github-actions-change == 'true' ||
+        steps.filter-identity.outputs.identity-frontend-change == 'true'
```

Comment:
we need filters on operate and tasklist client folders

Reply:
hups

Reply:
Sure, but why we have now two ways of doing it ?

Like https://github.com/camunda/camunda/blob/main/.github/actions/paths-filter/action.yml#L62-L69 

VS https://github.com/camunda/camunda/blob/main/.github/actions/paths-filter/action.yml#L55-L60

Reply:
> Sure, but why we have now two ways of doing it ?

Ah, they are functionally equivalent but I thought it might be nice to scope team specific filters (as long as needed) out into separate steps - for a logical partitioning.